src/dynapi/SDL_dynapi.sym

```diff
@@ -923,6 +923,23 @@ SDL3_0.0.0 {
     SDL_RWprintf;
     SDL_RWvprintf;
     SDL_AllocateEventMemory;
+    SDL_CleanupEvent;
```

Comment:
This doesn't exist and is causing build failures

Reply:
```suggestion
```